[docs] updating docs to include ai-powered ide guides (#1706)
coinbase/onchainkit: site/docs/pages/guides/using-ai-powered-ides.mdx

---
title: Use AI-powered IDEs · OnchainKit
description: How to use AI-powered IDEs to generate code for OnchainKit.
---

## Use AI-powered IDEs

> You can also directly download the [context](https://github.com/fakepixels/md-generator/blob/master/combined-ock-docs-0.35.8.mdx) and import it into your AI-powered IDE such as Cursor or Replit.

Reviewer: ```suggestion
You can download OnchainKit's [context](https://github.com/fakepixels/md-generator/blob/master/combined-ock-docs-0.35.8.mdx) and import it into AI-powered IDEs such as Cursor or Replit.
```

Imo `the your AI-powered IDE such as` is a bit awkward

> In addition, you can import a `.cursorrules` file in the root of your project via [Cursor Directory](https://cursor.directory/onchainkit). They also have an array of resources [here](https://cursor.directory/learn) on how to use AI-powered IDEs.

Reviewer: ```suggestion
In addition, you can import a `.cursorrules` file in the root of your project via the [Cursor Directory](https://cursor.directory/onchainkit). Cursor also has an [array of resources](https://cursor.directory/learn) on how to use AI-powered IDEs.
```

<img alt="Cursor"
  src="https://onchainkit.xyz/assets/cursor-dir.gif"
  height="364"/>

> ## AI Toolings

Reviewer: ```suggestion
## AI Tooling
```

### Replit
[Replit](https://replit.com) is a cloud-based coding platform that streamlines the process of setting up, building, sharing, and deploying projects. It allows developers to code in a Google Docs-like environment, and pre-built templates provide a great starting point for building a website, app, or game. Its new AI Agent can assist with the code development process and work with several files at once, making the programming process feel like a one-on-one conversation.

### Cursor

> [Cursor](https://cursor.com) is an AI-powered code editor that makes the programming experience feel like magic. Built as a fork of VS Code, it boasts powerful features like AI code completion, natural language editing, and codebase understanding. Cursor Pro is free for the first two weeks after signup, and offers more powerful models. We recommend starting your project on Replit and using this guide to open your project in Cursor.

Author: Removed that sentence. Thx for the catch
